packages/lib-classifier/src/store/Workflows.js

```diff
@@ -0,0 +1,38 @@
+import { getRoot, flow, types } from 'mobx-state-tree'
+import Workflow from './Workflow'
+
+const Workflows = types
+  .model('Workflows', {
+    current: types.maybe(types.reference(Workflow)),
+    workflows: types.optional(types.array(Workflow), [])
+  })
+
+  .actions(self => ({
+    afterAttach () {
+      self.fetchWorkflows()
+    },
+
+    fetchWorkflows: flow(function * fetchWorkflows () {
+      const { client, project } = getRoot(self)
+      const workflowIds = project.links.active_workflows
+
+      try {
+        const workflowFetchers = workflowIds.map(workflowId =>
```

Comment:
I know this might just be here for demo purposes, but this won't be a good strategy for workflow selection for several projects, Notes from Nature being the obvious one, since it has so many active workflows. In the current workflow selection code, default workflow is preferred, then a random id is selected out of the active workflow links. Then the workflow is only requested once the id is defined.